### DISCUSSION TITLE
is there anyway to access the position of a materialTopTab nav from the screens?

### DISCUSSION THREAD

**Discussion by kyle-belle** [react-navigation/react-navigation#9671] [Q&A]
Is there anyway to access the position (animated value) of a materialTopTab nav from the screens?
right now it seems it can only be accessed when rendering a custom tab bar but i'd like it in the screens as well

**Comment by pastean** [comment]
I've been struggling for a couple of days with this one as well. I tried using https://github.com/nandorojo/shared-animations to provide the position prop to multiple components, however it crashes with a reanimated error, so I think I will head in that direction.
